src/r_data.c

```diff
@@ -376,7 +376,7 @@ static void R_GenerateLookup(int texnum, int *const errors)
 	  const patch_t *realpatch = V_CachePatchNum(pat, PU_CACHE);
 	  int x, x1 = patch++->originx, x2 = x1 + SHORT(realpatch->width);
 	  const int *cofs = realpatch->columnofs - x1;
-	  
```

Comment:
Could you please remove the whitespace changes? There are a lot of formatting issues in the MBF code, so we have decided to leave it as it is to minimise diffs.

Reply:
Oh oops I didn't notice that happened, will do.